pom.xml

```diff
@@ -221,6 +221,12 @@
                 <groupId>io.jaegertracing</groupId>
                 <artifactId>jaeger-client</artifactId>
                 <version>${jaeger.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>io.jaegertracing</groupId>
+                        <artifactId>jaeger-thrift</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
I'm not sure exclusions apply in dependencyManagement

Reply:
When I add this exclusion in the dependency management IntelliJ removes it from the External Libraries list, so I assumed it worked.

Reply:
Maybe, I honestly don't know :)  I'd trust more a maven command to pull the dependencies rather than the IntelliJ plugin (which may or not be invoking maven underlying, I don't think it is)

Reply:
I just checked with `mvn dependency:tree` and the exclusion in dependency management removes the artifact from the tree.